docker/Dockerfile.chronograf.prod

```diff
@@ -38,17 +29,20 @@ ARG INJECT_HEADER
 # Injecting strings into the html never went wrong. we use this for google tag manager
 ARG INJECT_BODY
 
-ENV PATH="/node/bin:${PATH}"
-RUN mkdir /node && \
-    curl -sL https://git.io/n-install | N_PREFIX=/node bash -s -- -q
-
 COPY . /repo/ui
 WORKDIR /repo/ui
 
+RUN apk \
+    --update \
+    --no-cache \
+    --virtual build-dependencies \
+    add \
+    git
+
 # these are all run together as docker's caching mechanism
 # makes big steps like yarn install expensive
-RUN npx yarn install --production=false && \
-    npx yarn generate && \
+RUN yarn install --production=false && \
```

Comment:
wouldn't we need to install yarn prior to this command?

Reply:
the alpine node image has yarn pre-installed :tada: